src/deepsparse/server/server.py

```diff
@@ -208,42 +206,32 @@ def _add_endpoint(
     endpoint_config: EndpointConfig,
     executor: ThreadPoolExecutor,
     context: Context,
-    pipeline_logger: ManagerLogger,
+    server_logger: BaseLogger,
 ):
     pipeline_config = endpoint_config.to_pipeline_config()
     pipeline_config.kwargs["executor"] = executor
 
     _LOGGER.info(f"Initializing pipeline for '{endpoint_config.name}'")
-    pipeline = Pipeline.from_config(pipeline_config, context)
+    pipeline = Pipeline.from_config(pipeline_config, context, server_logger)
```

Comment:
i think this is fine for now, but still think we should have separate loggers for each pipeline. can do in a separate PR